.github/ISSUE_TEMPLATE/flaky-test-report.yml

```diff
@@ -66,7 +66,7 @@ body:
   - type: textarea
     attributes:
       label: Parent issue
-      value: "#1733"
+      value: ""
```

Comment:
`value` must be of type String and cannot be empty. @kaijchen

Reply:
Thanks, can you submit a PR to fix it?
Maybe let's just remove this field?

Reply:
ok